scripts/setup-local-environment.sh

```diff
@@ -5,6 +5,7 @@ INCLUDE_BASE=${INCLUDE_BASE:-true} # INCLUDE_UTILS must be true to work.
 INCLUDE_UTILS=${INCLUDE_UTILS:-false}
 INCLUDE_DEVELOPER=${INCLUDE_DEVELOPER:-false}
 
+PANDIR=/var/panoptes
```

Comment:
In #1001 I actually have this set with the default of PANDIR otherwise use the current dir.  But this relies somewhat on the env vars being set up correctly.

```suggestion
PANDIR="${PANDIR:-$PWD/../}"
```